modal/functions.py

```diff
@@ -644,6 +646,7 @@ class CloudProvider(Enum):
 
 
 def parse_cloud_provider(value: str, gpu_config: api_pb2.GPUConfig) -> "api_pb2.CloudProvider.V":
+    """mdmd:hidden"""
```

Comment:
Perhaps can be `_parse_cloud_provider` private? cc @aksh-at

Reply:
ah yes good catch, it should be hidden